server/routes/prisonerRoutes.ts

```diff
@@ -20,7 +20,7 @@ export default function prisonerRoutes({
           data.pipe(res)
         })
         .catch(() => {
-          const placeHolder = path.join(process.cwd(), '/assets/images/image-missing.jpg')
+          const placeHolder = path.join(process.cwd(), '/dist/assets/images/image-missing.jpg')
```

Comment:
latest esbuild stuff means that the assets are under `dist` on the server.  This is different from normally just displaying the image since we're loading this one from the file system instead in node